### ISSUE TITLE
Support for chai 2.0.0

### ISSUE THREAD

**Issue by scottohara** [domenic/chai-as-promised#82]
Peer dependencies in CaP 4.1.1 are currently restricted to chai 1.x:
```
"peerDependencies": {
    "chai": ">= 1.7.0 < 2"
},
```

Chai 2.0.0 was released recently, but trying to update using `npm update chai` with a package.json containing:
```
"devDependencies": {
    "chai": "^2.0.0",
    "chai-as-promised": "^4.1.1",
```

yields:

```
npm ERR! peerinvalid The package chai does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer chai-as-promised@4.1.1 wants chai@>= 1.7.0 < 2
```

**Comment by markthethomas** [comment]
:+1:

**Comment by demisx** [comment]
+1 Ple-e-e-e-ase.

**Comment by domenic** [comment]
Every +1 comment makes me delay this an extra day. Locking.

**Comment by domenic** [comment]
Fixed by 0bae928b72683b07c2b82b7974f71c500b5a2451, release coming soon, maybe with some more bug fixes.
